Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -56,7 +56,7 @@ public class SchemeGenerator {
                     let debugConfig = suitableConfig(for: .debug, in: project)
                     let releaseConfig = suitableConfig(for: .release, in: project)
 
-                    let scheme = Scheme(
+                    let scheme = Scheme.init(
```

Comment:
We can drop this

Reply:
True I just used it to get Xcode to show me some proper code completions :D
```suggestion
                    let scheme = Scheme(
```